src/main/java/core/mate/academy/impl/TruckProducer.java

```diff
@@ -0,0 +1,19 @@
+package core.mate.academy.impl;
+
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
```

Comment:
The class implementation should not begin with an empty line. Please remove the empty line before the class declaration.